src/Illuminate/Cache/DatabaseLock.php

```diff
@@ -44,6 +44,7 @@ class DatabaseLock extends Lock
      * @param  int  $seconds
      * @param  string|null  $owner
      * @param  array  $lottery
+     * @param  int  $defaultTimeoutInSeconds
```

Comment:
Ich gehe mal davon aus, dass die Zahl nicht negativ sein kann:
```suggestion
     * @param  non-negative-int  $defaultTimeoutInSeconds
```